SmartDeviceLink/public/SDLGetVehicleData.h

```diff
@@ -178,12 +178,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nullable, strong, nonatomic) NSNumber<SDLBool> *climateData;
 
 /**
- * The external temperature in degrees celsius. This parameter is deprecated starting RPC Spec 7.1, please see climateData.
+ * The external temperature in degrees celsius.
  *
- * @deprecated in SmartDeviceLink 7.1.0
  * @added in SmartDeviceLink 2.0.0
  */
-@property (nullable, strong, nonatomic) NSNumber<SDLBool> *externalTemperature __deprecated_msg("Use climateData instead on 7.1+ RPC version connections");
+@property (nullable, strong, nonatomic) NSNumber<SDLBool> *externalTemperature;
```

Comment:
This parameter, `externalTemperature`, should be deprecated here and in the other vehicle data class as it was in previous reviews. Just so there is no confusion, the `SDLRPCParameterNameExternalTemperature` does not need to be deprecated because the same parameter name is now being used in the new `ClimateData` struct.

Reply:
I see, I got it wrong. Already fixed.